app/views/frame-connect.jade

```diff
@@ -10,6 +10,8 @@ div(ng-controller="AuthCtrl")
                 h3 Connect to Neo4j
                 p.lead
                   | Database access requires an authenticated connection.
+                p.warn(ng-show="neo4j.clusterRole && failed_connection_attempts > 2")
+                  | Note: User management events do not propagate when running a core-edge cluster. Does your user exist on this server instance?
```

Comment:
I'd suggest avoiding direct mention of "core-edge" and simplifying the language. What about:

> Recall that user management events do not propagate across a cluster. Does your user exist here?